src/form/ColorMapInput.tsx

```diff
@@ -235,6 +267,31 @@ const ColorMapInput: React.FC<ColorMapProps & InjectedIntlProps> = (props) => {
       htmlFor={name}
       className={formStyles.Label}
     >
+      {showCustomColormapModal? 
+        // <Modal
+        //   title={'CUSTOM COLORMAP'}
+        // >
+        <ModalBackground
+          title={'CUSTOM COLORMAP'}
+          handleClose={() => setShowCustomColormapModal(false)}
+          // value depends on content, but should be precisely hardcoded
+          height={'816px'} 
```

Comment:
You can also give value in percentage (i.e. 60%), not only in pixels. There is a default value for the modal as well (70% width and height of the whole screen). So if you don't pass anything to width and height, it will be 70% width and height.

Reply:
Gave the width and height in percentage